src/XrdSciTokens.cmake

```diff
@@ -31,6 +33,22 @@ target_include_directories(
    XrdSciTokens/vendor/picojson
    XrdSciTokens/vendor/inih )
 
+if( BUILD_MACAROONS )
+   target_include_directories(
+      ${LIB_XRD_SCITOKENS}
+      PRIVATE
+      ${MACAROONS_INCLUDES} )
+
+   target_compile_definitions(${LIB_XRD_SCITOKENS} PRIVATE HAVE_MACAROONS)
+
+   target_link_libraries(
+       ${LIB_XRD_SCITOKENS}
+       PRIVATE
+       uuid::uuid
+       OpenSSL::Crypto
+       ${MACAROONS_LIB} )
+endif()
```

Comment:
I'm not a big fan of the mixing up between scitokens and macaroons introduced here. Is there no way to do this in a way that doesn't require such invasive changes?

Reply:
Maybe I can come up with a more directed approach for this. I agree that given the scale of change here we really need to run regression tests before we let this out.